### PR TITLE
Update README examples

### PR DESCRIPTION
**Pull request by rtgdk**
The commands did not work properly if the folder is not specified correctly.

### CODE REVIEW

**Comment by pombredanne** [comment]
@rtgdk Thanks. let me review this

File: README.md~ [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Python SPDX Library to parse, validate and create SPDX documents
```
Comment (pombredanne) [inline README.md~]: This file should not have been committed.

File: README.md [hunk 2]
```diff
@@ -57,22 +57,122 @@ Example tag/value parsing usage:
 The `examples` directory contains several code samples:
 
 * `parse_tv.py` is an example tag/vlue parsing uage.
-  Try running `python parse_tv.py 'data/SPDXSimpleTag.tag' `
+  Try running `python parse_tv.py '../data/SPDXSimpleTag.tag' `
```
Comment (pombredanne) [inline README.md]: Are you sure?

File: README.md [hunk 3]
```diff
@@ -57,22 +57,122 @@ Example tag/value parsing usage:
...
 * `rdf_to_tv.py` demonstrates how to convert an RDF file to a tag/value one.
-   To test it run `python rdf_to_tv.py data/SPDXRdfExample.rdf converted.tag`
+   To test it run `python rdf_to_tv.py ../data/SPDXRdfExample.rdf converted.tag`
 
 * `pp_rdf.py` demonstrates how to pretty-print an RDF file, to test it run 
-  `python pp_rdf.py data/SPDXRdfExample.rdf pretty.rdf`
+  `python pp_rdf.py ../data/SPDXRdfExample.rdf pretty.rdf`
+
+
+# Installation
```
Comment (pombredanne) [inline README.md]: Why did you copy this block from above here down? I am confused.

File: README.md (hunk 2 above)
Comment (rtgdk) [inline README.md]: Yes. For running this sample code either you have to be in `examples` directory and then try `python parse_tv.py '../data/SPDXSimpleTag.tag'` or in the `main` directory and then try `python examples/parse_tv.py 'data/SPDXSimpleTag.tag'` .

Comment (pombredanne) [inline README.md]: ok

**Comment by pombredanne** [comment]
Can you squash these commits?

**Comment by rtgdk** [comment]
@pombredanne  Commits squashed.

**Comment by pombredanne** [comment]
@rtgdk Thanks. Let me review and merge. @sschuberth would you know why travis and appevyor runs are not complete? I did not check the details though.

**Comment by sschuberth** [comment]
@pombredanne No idea why Travis CI and AppVeyor did not report back their statuses, as they seem to have [build](https://travis-ci.org/spdx/tools-python/pull_requests) [something](https://ci.appveyor.com/project/spdx/tools-python/history), but I'm not seeing commit 4064a2b in there... let's see if this happens again for the next PR.

**Comment by pombredanne** [comment]
@rtgdk Thanks!

**Pull request merged**